app.js

```diff
@@ -20,9 +20,27 @@ app.allUsers = [];
 // Array of usernames queued up to be assigned to upcoming new issues
 app.queuedUsers = [];
 
-// When receiving a POST request from Sentry:
-app.post("/", async function(request, response) {
+const errorWrapper = fn => {
+  return async (req, res, next) => {
```

Comment:
@nolachen what happened to using `next` so we could use the global error handler?

Reply:
@scefali we tried to use `next` but then in the global error handler, the `res.sentry` to access the error ID no longer works (it's undefined). I guess the express sdk doesn't pass the error ID along? And we can't rely on `lastEventID` since it's async. Not sure if there's a better way but this seems simpler for now